### PR TITLE
Aut 33 refactoring test structure

### PR DESCRIPTION
**Pull request by igorlg**
Refactored test structure.

### CODE REVIEW

File: test/unit/helpers/spec_helper.rb [hunk 1]
```diff
@@ -16,11 +16,16 @@
...
               [
-                # { platform: 'redhat',  version: '7.2'      },
-                { platform: 'suse',    version: '12.1'     },
-                # { platform: 'amazon',  version: '2016.03'  },
-                { platform: 'windows', version: '2012R2'   }
+                { platform: 'amazon', version: '2016.03' }
+                # { platform: 'redhat',  version: '7.2'     },
+                # { platform: 'suse',    version: '12.1'    },
+                # { platform: 'amazon',  version: '2016.03' },
+                # { platform: 'windows', version: '2012R2'  },
```
Comment (shinitiandrei) [inline test/unit/helpers/spec_helper.rb]: do we need to comment them all?

File: test/unit/helpers/spec_helper.rb [hunk 2]
```diff
@@ -16,11 +16,16 @@
...
               ]
             end
 
+RSpec.configure do |c|
+  c.log_level = :error # unless ENV.include? 'TRAVIS'
+end
+
```
Comment (shinitiandrei) [inline test/unit/helpers/spec_helper.rb]: Cool!! just really errors showing up

File: test/unit/helpers/spec_helper.rb (hunk 1 above)
Comment (igorlg) [inline test/unit/helpers/spec_helper.rb]: Probably not, let's just uncomment, now that the unit tests are running a bit faster...

Comment (igorlg) [inline test/unit/helpers/spec_helper.rb]: @shinitiandrei done! =]

**Pull request merged**